### PR TITLE
More bindings

### PR DESCRIPTION
**Pull request by accumulator**

### CODE REVIEW

**Comment by rinigus** [comment]
I'll have to read it, but here is the first comment:

All properties were ordered by Osmo alphabetically. So, if Height and Width are even close to each other from concept POV, Large will be in between. Let's keep the same order, at least for now. 

But let me read the code before any changes...

**Comment by rinigus** [comment]
... I'll have to postpone the review for a day or two, sorry

File: qml/AttributionButton.qml [hunk 1]
```diff
@@ -40,16 +48,5 @@ IconButtonPL {
...
 
-    property string logo: ""
-
     onClicked: app.showMenu("AttributionPage.qml");
-    onLogoChanged: attributionButton.icon.source = logo ?
-        app.getIcon("icons/attribution/%1".arg(logo)) : "";
-
-    Connections {
-        target: app.styler
-        onIconVariantChanged: attributionButton.icon.source = attributionButton.logo ?
```
Comment (rinigus) [inline qml/AttributionButton.qml]: `app.getIcon` depends on the iconVariant of the styler. This allows us to switch from light to the dark maps and change all icons accordingly. I expect that such binding rewrite will kill this property. Have you tried to swap between light/dark maps?

Comment (accumulator) [reply qml/AttributionButton.qml]: Yes, this works. The iconvariant changes at the same time the logo does, so a logo change will always pick up the iconvariant.

Comment (rinigus) [reply qml/AttributionButton.qml]: Just to be clear - its in some aspects a toss of a coin in this case. Its either we are strict and keep defined signals  (as before) in this case OR we write a comment regarding this in Map.qml. 

Taking into account that the application is already complex (while binding does reduce it!), I am inclined to be careful and go for the option 1 (keep signaling over here). What would be your call taking into account my arguments?

**Pull request closed without merging**